### PR TITLE
update links/variables for polkadot v1.0.1

### PR DESCRIPTION
**Pull request by eshaben**
Goes with EN PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/831

See that PR description for more info.

### CODE REVIEW

File: builders/interoperability/xcm/fees.md [hunk 1]
```diff
@@ -60,18 +60,28 @@ Substrate已推出一个权重系统，决定一个函数的权重，也就是
 
 ### Polkadot {: #polkadot }
 
-如同先前提到的，波卡目前对所有XCM指令采取[固定权重数量](https://github.com/paritytech/polkadot/blob/{{networks.polkadot.spec_version}}/runtime/polkadot/src/xcm_config.rs#L111){target=_blank}的计算方式，也就是每条指令`{{ networks.polkadot.xcm_instructions.weight.display }}`权重单位。
+The total weight costs on Polkadot take into consideration database reads and writes in addition to the weight required for a given instruction. Polkadot uses benchmarked weights for instructions and database read and write operations. The breakdown of weight costs for the database operations is as follows:
```
Comment (themacexpert) [inline builders/interoperability/xcm/fees.md:63]: Assume this will be translated in the future

**Review by themacexpert — APPROVED** [review]
LGTM. The formatting for the below section on the XCM fees page was a bit large, it might just be me. I tried it in 2 different browsers and got the same thing. 

Screenshot 2024-01-08 at 12 06 18 PM

File: builders/interoperability/xcm/fees.md (hunk 1 above)
Comment (eshaben) [reply builders/interoperability/xcm/fees.md:63]: oops forgot about this!

@wuzhong-papermoon can you please translate this?

File: builders/interoperability/xcm/xc20/xtokens.md [hunk 2]
```diff
@@ -73,17 +73,17 @@ X-Tokens Pallet包含以下只读函数以获取pallet常量：
 
 ## XCM Instructions for Transfers via X-Tokens {: #xcm-instructions }
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/xtokens.md:74]: @wuzhong-papermoon could you also translate this title please?

**Comment by eshaben** [comment]
@themacexpert That is super weird! I'm not seeing it. I merged in some stuff before trying, but I don't think that would impact the results. Did you try spinning up the CN site and using this branch `eshaben/polkadot-runtime-v1.0.1` for the EN repo at the same time? Not really sure why that would impact it either but just spitballing here haha 😅

**Comment by themacexpert** [comment]
> @themacexpert That is super weird! I'm not seeing it. I merged in some stuff before trying, but I don't think that would impact the results. Did you try spinning up the CN site and using this branch `eshaben/polkadot-runtime-v1.0.1` for the EN repo at the same time? Not really sure why that would impact it either but just spitballing here haha 😅

I did originally built it with both the c and e sites pointing to same but I rebuilt with a new series of repos and it's all good now. 👍

**Pull request merged**